CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+Contributing to WildFly Elytron
+==================================
+
+### Welcome to the WildFly Elytron project! We welcome contributions from the community. This guide will walk you through the steps for getting started on our project.
+
+- [Forking the Project](#forking-the-project)
+- [Issues](#issues)
+  * [Good First Issues](#good-first-issues)
+- [Setting up your Developer Environment](#setting-up-your-developer-environment)
+- [Contributing Guidelines](#contributing-guidelines)
+- [Community](#community)
+
+
+## Forking the Project 
+To contribute, you will first need to fork the [wildfly-elytron](https://github.com/wildfly-security/wildfly-elytron) repository. 
+
+This can be done by looking in the top-right corner of the repository page and clicking "Fork".
+![fork](assets/images/fork.jpg)
+
+The next step is to clone your newly forked repository onto your local workspace. This can be done by going to your newly forked repository, which should be at `https://github.com/USERNAME/wildfly-elytron`. 
+
+Then, there will be a green button that says "Code". Click on that and copy the URL.
+
+![clone](assets/images/clone.png)
+
+Then, in your terminal, paste the following command:
+`git clone [URL]`
```

Comment:
To get this to appear on a new line, I think you could use this:

Then, in your terminal, paste the following command:

```
git clone [URL]
```

Be sure to replace [URL] with the URL that you copied above.

Reply:
Updated